libs/wordpress-plugin/plugin/trunk/block/edit-or-preview/edit/service-callbacks.ts

```diff
@@ -33,7 +39,7 @@ export const callService = async ({
   });
 
   if ("error" in apiResponse) {
-    let errorMessage = apiResponse.error ?? "An unknown error occured";
+    let errorMessage = apiResponse.error ?? "An unknown error occurred";
```

Comment:
Apologies if I'm missing something obvious, but why would the error message ever be `apiResponse.error`? We're nesting the error message in an `errors` array in the BP API if I recall correctly.

Reply:
This the WordPress API, where we only extract a single error (or at least we do in addition to providing the original BP response) – see [here](https://github.com/blockprotocol/blockprotocol/blob/f4aad130c59e3d2c8bdcf154322bd1b1dd9214fa/libs/wordpress-plugin/plugin/trunk/server/block-api-endpoints.php#LL649)